packages/communication-react/review/communication-react.api.md

```diff
@@ -1942,6 +1942,7 @@ export type MessageThreadProps = {
     disableEditing?: boolean;
     strings?: Partial<MessageThreadStrings>;
     fileDownloadHandler?: FileDownloadHandler;
+    dateTimeFormat?: (messageDate: Date) => string;
```

Comment:
How about we go 1 step further?
Since Javascript doesn't have native date formatting using `YYYY, MM, DD` etc.
We can use a popular library like momentjs, datejs behind the scenes and accept a date time format making the implementation standardized and inline with other programming languages

Reply:
OR use the native platform supported formatting features like
![image](https://user-images.githubusercontent.com/9782747/167509269-96a6cb06-1550-4959-b305-3f1bb7cbf9b1.png)

Reply:
Here's an article to help you out
https://dockyard.com/blog/2020/02/14/you-probably-don-t-need-moment-js-anymore